### PR TITLE
Improve/ CLI selected db index

### PR DESCRIPTION
**Pull request by ArtyomPodymov**
* #RI-2122 - update messages
* #RI-2122 - display selected db index in cli
* #RI-2122 - create a new client if the previous one was closed

### CODE REVIEW

**Review by romansergeenkosofteq — APPROVED** [review]

File: redisinsight/ui/src/slices/cli/cli-output.ts [hunk 1]
```diff
@@ -224,3 +255,20 @@ export function processUnrepeatableNumber(
     onSuccessAction?.()
   }
 }
+handleRecreateClient
```
Comment (zalenskiSofteq) [inline redisinsight/ui/src/slices/cli/cli-output.ts:258]: I think it needs to be removed

File: redisinsight/ui/src/components/cli/components/cli-input/CliInputWrapper.tsx [hunk 2]
```diff
@@ -6,6 +6,7 @@ import { appRedisCommandsSelector } from 'uiSrc/slices/app/redis-commands'
 import CliAutocomplete from './CliAutocomplete'
 
 import CliInput from './CliInput'
+import { outputSelector } from 'uiSrc/slices/cli/cli-output';
```
Comment (zalenskiSofteq) [inline redisinsight/ui/src/components/cli/components/cli-input/CliInputWrapper.tsx:9]: Could you move this absolute path import before relative imports?

File: redisinsight/ui/src/utils/cliHelper.tsx [hunk 3]
```diff
@@ -121,6 +121,18 @@ const checkUnsupportedModuleCommand = (loadedModules: RedisModuleDto[], commandL
   return commandModule
 }
 
+const getDbIndexFromSelectQuery = (query: string): number => {
+  const [command, ...args] = query.trim().split(' ')
+  if (command.toLowerCase() !== 'select') {
```
Comment (zalenskiSofteq) [inline redisinsight/ui/src/utils/cliHelper.tsx:126]: May be better use exported constant `SelectCommand` from `cli-output` slice?

File: redisinsight/ui/src/slices/cli/cli-output.ts (hunk 1 above)
Comment (ArtyomPodymov) [reply redisinsight/ui/src/slices/cli/cli-output.ts]: thanks, done

File: redisinsight/ui/src/components/cli/components/cli-input/CliInputWrapper.tsx (hunk 2 above)
Comment (ArtyomPodymov) [reply redisinsight/ui/src/components/cli/components/cli-input/CliInputWrapper.tsx]: done

File: redisinsight/ui/src/utils/cliHelper.tsx (hunk 3 above)
Comment (ArtyomPodymov) [reply redisinsight/ui/src/utils/cliHelper.tsx]: done

**Review by zalenskiSofteq — APPROVED** [review]

**Pull request merged**